### PR TITLE
Feat/chatfrompost 4.2

### PR DESCRIPTION
**Pull request by Evtaft22**
- message a user from their post

- if there is already a chat between the current user and the user who created the post, this message will be added to that chat

- if there is not and existing chat, one will be created and the message will be added

### CODE REVIEW

**Review by bsbrionesOS — APPROVED** [review]

File: client/src/components/PostFullMessage.jsx [hunk 1]
```diff
@@ -105,7 +154,7 @@ const PostFullMessage = ({ match, menu, currentUser }) => {
         </div>
         <div style={{ marginTop: "160px" }}>
           <div style={{ fontSize: "18px" }}>{post.message}</div>
```
Comment (jstrzesz) [inline client/src/components/PostFullMessage.jsx:156]: move styling to css file, no inline styling

File: client/src/components/PostFullMessage.jsx [hunk 2]
```diff
@@ -105,7 +154,7 @@ const PostFullMessage = ({ match, menu, currentUser }) => {
         </div>
         <div style={{ marginTop: "160px" }}>
```
Comment (jstrzesz) [inline client/src/components/PostFullMessage.jsx:155]: move styling to css file, no inline styling

File: client/src/components/PostFullMessage.jsx [hunk 3]
```diff
@@ -105,7 +154,7 @@ const PostFullMessage = ({ match, menu, currentUser }) => {
...
           <div style={{ fontSize: "18px" }}>{post.message}</div>
-          <div style={{ fontSize: "16px", marginTop: "10px" }}>{postTags.join("   ")}</div>
+          <div style={{ fontSize: "16px", marginTop: "10px" }}>{postTags.join(" ")}</div>
```
Comment (jstrzesz) [inline client/src/components/PostFullMessage.jsx:157]: move styling to css file, no inline styling

File: client/src/components/PostFullMessage.jsx [hunk 4]
```diff
@@ -139,7 +188,7 @@ const PostFullMessage = ({ match, menu, currentUser }) => {
             type="text"
             placeholder="Message"
           />
-          <button onClick={sendMessage} style={{ marginLeft: "5px", backgroundColor: "#eb8c34" }}>
+          <button onClick={onSubmit} style={{ marginLeft: "5px", backgroundColor: "#eb8c34" }}>
```
Comment (jstrzesz) [inline client/src/components/PostFullMessage.jsx:191]: move styling to css file, no inline styling

**Review by HenokSamuel — APPROVED** [review]

**Pull request merged**